cli/bblfshctl/cmd/driver_install.go

```diff
@@ -133,7 +133,7 @@ func (c *DriverInstallCommand) Execute(args []string) error {
 
 func (c *DriverInstallCommand) Validate() error {
 	if !c.All && !c.Recommended && (c.Args.Language == "" || c.Args.ImageReference == "") {
-		return fmt.Errorf("error --language and --image are mandatory")
+		return fmt.Errorf("error `language` and `image` arguments are mandatory")
```

Comment:
Maybe "positional arguments" instead of "arguments" to make it clearer what they are?

Reply:
Good point! Will update shortly.